### PR TITLE
details: update link for reporting license issues

### PR DESCRIPTION
**Pull request by Staudey**
## Description
This updates the "Report license issues" link on the package details page from phabricator to the new issues.getsol.us

## Submitter Checklist

[x] Squashed commits with `git rebase -i` (if needed)  
[x] Built solus-sc and verified that the patch worked (if needed)

### CODE REVIEW

File: solus_sc/details.py [hunk 1]
```diff
@@ -442,7 +442,7 @@ def setup_license_view(self):
         self.view_stack.add_titled(
             lic_wrap, "license", _("License"))
 
-        reportURI = "https://dev.getsol.us/maniphest/task/edit/form/1/"
+        reportURI = "https://github.com/getsolus/packages/issues/new/"
```
Comment (davidjharder) [inline solus_sc/details.py:445]: This can be `issues.getsol.us`

Comment (Staudey) [reply solus_sc/details.py]: Man, I knew there was an URL like that, but I couldn't think of the right one when doing this. Fixed!

**Comment by davidjharder** [comment]
LGTM, but I won't merge as I'm not too familiar with the Software Center

**Comment by Staudey** [comment]
@ermo also mentioned that the phab.getsol.us instance wasn't really meant for public consumption, so there's that as well.

**Comment by Staudey** [comment]
Closing in favour of #251

**Comment by Staudey** [comment]
Actually the issue URL part is still useful/necessary. Gonna rework this accordingly.

**Pull request merged**